phantomgames/Search/SearchGameRepository.swift

```diff
@@ -58,4 +58,3 @@ class SearchGameRepository: SearchGameRepositoryType {
         dataTask.resume()
     }
```

Comment:
If the "request" and "call" functions are used else where in the code, You can create a URL session Extension